README.md

```diff
@@ -1,5 +1,21 @@
 # netbeans-l10n
```

Comment:
We should be able to add the license header the same as XML.

Reply:
header added.